test/examples/ucp_client_server.c

```diff
@@ -101,12 +101,12 @@ static void err_cb(void *arg, ucp_ep_h ep, ucs_status_t status)
 /**
  * Set an address for the server to listen on - INADDR_ANY on a well known port.
  */
-void set_listen_addr(struct sockaddr_in *listen_addr)
+void set_listen_addr(const char *address_str, struct sockaddr_in *listen_addr)
 {
     /* The server will listen on INADDR_ANY */
     memset(listen_addr, 0, sizeof(struct sockaddr_in));
     listen_addr->sin_family      = AF_INET;
-    listen_addr->sin_addr.s_addr = INADDR_ANY;
+    listen_addr->sin_addr.s_addr = (address_str) ? inet_addr(address_str) : INADDR_ANY;
```

Comment:
should we handle IPv6? IMO we have to check sa_family at least

Reply:
@evgeny-leksikov I'm in favor of this. Should we handle it in this PR or a follow-up? Either works for me. Lmk

Reply:
@Akshay-Venkatesh up to you. As far as I understand, in current state, there is a bug if pass IPv6 address, right?

Reply:
@Akshay-Venkatesh IMHO it's better to handle both case in this PR.

Reply:
Sorry for the late reply I was OOTO. I'll add code to handle both cases soon.